### ISSUE TITLE
Pi Pico PIO Example does nothing

### ISSUE THREAD

**Issue by zoggins** [pschatzmann/logic-analyzer#12]
I have done everything that I think is correct, but there is just no data coming out of the serial port.  Any help would be appreciated.  Thanks.

**Comment by pschatzmann** [comment]
I haven't used this for a long time. 
Did you follow [these steps](https://github.com/pschatzmann/logic-analyzer#connecting-to-pulseview) ?

**Comment by zoggins** [comment]
Yes.  I have done those instructions on multiple Picos and multiple computers and the device never shows up when i hit scan.  The Pico seems to be a some stuck state because the Arduino UI can't even force it to programming mode, I have to manually reboot the board to program it again.  I have also tried with the  testData function and real signals, all the same result.

**Comment by zoggins** [comment]
Its no big deal I will either debug it myself or try something else.

**Comment by zoggins** [comment]
It appears its something with the serial output.  I can hit it directly with a terminal for a little bit and then the serial communication will lock up.

**Comment by pschatzmann** [comment]
Did you try to reduce the Serial speed ?

**Comment by pschatzmann** [comment]
This should now be fixed by 7aa0c6b — same root cause as #3: `PicoCapturePIO::arm()` never called `pio_gpio_init()` / `pio_sm_set_consecutive_pindirs()`, so the PIO block wasn't actually connected to the GPIO pins, meaning the capture always read nothing regardless of the signal on the pins.

I also fixed:
- the SUMP sample byte order (was being sent big-endian instead of the little-endian PulseView/libsigrok expects, see #2)
- the `logic-analyzer-pico` example's `#ifndef PICO` guard, which isn't defined on Pico 2/2W boards (use `ARDUINO_ARCH_RP2040`)

Tested end-to-end on a Raspberry Pi Pico 2 W: compiled and flashed `logic-analyzer-pico-pio`, and `sigrok-cli --driver ols:conn=/dev/ttyACM0 --scan` now detects the device and captures the requested number of samples cleanly. Please pull the latest main and retest on your Pico — let me know if it's still not producing data.
